webapp/src/container.tsx

```diff
@@ -249,6 +249,7 @@ export class SettingsMenu extends data.Component<SettingsMenuProps, SettingsMenu
 
     signOutGithub() {
         pxt.tickEvent("menu.github.signout");
+        (document.querySelector(".more-dropdown-menuitem") as HTMLElement).blur();
```

Comment:
Mixing direct DOM manipulation with React should be avoided if what you want to do can be done in pure React. In this case you should be able to get a reference to React's instantiation of the dropdown menu and call `hide` on that. There are a bunch of ways to get and maintain a ref. Here's one way:

```
return <sui.DropdownMenu ..... ref={ref => this.dropdown = ref}>
```

Then, add a `hide` method to `SettingsMenu`:

```ts
    hide() {
        this.dropdown?.hide();
    }
```